source/distributed/cell_weights.cc

```diff
@@ -143,19 +143,20 @@ namespace parallel
         case Triangulation<dim, spacedim>::CELL_COARSEN:
           {
             std::set<unsigned int> fe_indices_children;
-            for (unsigned int child_index = 0;
-                 child_index < GeometryInfo<dim>::max_children_per_cell;
```

Comment:
why did you need to change this? it's compile-time know, i would keep it.

Reply:
I changed that line in case we want to use anisotropic refinement.

I know that the `p::d::Triangulation` class does not support anisotropic refinement currently, but whenever we decide to use anisotropic refinement sometime in the future (when `p4est` supports it), we will look after those like needles in a haystack.

Reply:
Good find. I think you made the right call!